src/ip.py

```diff
@@ -727,17 +727,24 @@ def do_neigh_flush(argv, af):
 def main(argv):
     # Detect Address family
     af = -1  # default / both
-    if argv and argv[0] == "-6":
+    if "-6" in argv:
         af = 6
-        argv.pop(0)
-    elif argv and argv[0] == "-4":
+        argv.remove("-6")
+    elif "-4" in argv:
         af = 4
-        argv.pop(0)
+        argv.remove("-4")
+    
+    # Ignore color option (-c --color)
+    for option in ["-c", "--c"]:
+        for arg in argv:
+            if arg.startswith(option):
+                perror("!Color option not available!")
+                argv.remove(arg)
 
     if not argv:
         return False
 
-    if argv[0] in ["-V", "-Version"]:
+    if argv[0] in ["-V", "-Version", "--version"]:
```

Comment:
`--version` doesn't seem to be supported by iproute2, can you revert this change? Or what is your rationale?

```bash
user@hostname:~$ ip --version
Option "-version" is unknown, try "ip -help".
user@hostname:~$ ip --Version
ip utility, iproute2-ss190107
```

Reply:
Thank you for the extensive comments. The rationale for including `--version` (as well as not taking into account the order of the options) is to make it easier to use. 

There is an argument against this with it being confusing when shifting from Mac to Linux when some commands don't transfer one-on-one. If the goal is to match the behaviour exactly I will update the PR (and I could also add it to the contributing guidelines?)

Reply:
With iproute2mac I am aiming for as much as identical behavior and flag compatibility with iproute on linux as possible. So here I would rather match the flag exactly. Thanks for understanding. 

And yes we should include this note in the contributing guidelines, after you PR I will extend them.